### PR TITLE
The Community Hairstyle Pack

### PR DESCRIPTION
**Pull request by DrMacCool**
## About The Pull Request

This PR is the product of a fun little project over on the [Beestation Forums](https://forums.beestation13.com/t/new-hairstyles/15009). It contains 18 new hairstyles and beards made up of personal additions and community requests.

**Additions w/ Credits**

**Hairstyles**
<details>
<summary> 'Bald 2', requested by Archanial </summary>

image

</details>
<details>
<summary> 'Curtains' </summary>

image

</details>
<details>
<summary>'Gelled Spikes'</summary>

image

</details>
<details>
<summary>'Moustache', requested by Engelwood</summary>

image

</details>
<details>
<summary>'Mohawk (Reverse)', requested by Engelwood</summary>

image

</details>
<details>
<summary>'Mullet', requested by Spockye</summary>

image

</details>
<details>
<summary>'Parted 2'</summary>

image

</details>
<details>
<summary>'Pompadour (Huge)', requested by Engelwood</summary>

image

</details>
<details>
<summary>'Ponytail (Alchemist)'</summary>

image

</details>
<details>
<summary>'Ponytail (Tight)'</summary>

image

</details>
<details>
<summary>'Short Afro'</summary>

image

</details>
<details>
<summary>'Spamton', requested by Crow004</summary>

image

</details>
<details>
<summary>'Pride', requested by Kapu1178</summary>

image

</details>

**Facial Hair**
<details>
<summary>'Eyebrows'</summary>

image

</details>
<details> <summary>'Beard ( Powerful)', requested by Dog132 </summary>

image
 
</details>
<details>
<summary>'Beard (The Colonel)', requested by Colonel</summary>

image

</details>
<details>
<summary>'Moustache (Robotnik)', requested by Engelwood</summary>

image

</details>
<details>
<summary>'Moustache ('Stache 'n Chops)', requested by Taylor</summary>

image

</details>

## Why It's Good For The Game

More customisation options for humans/ethereals/oozelings.

## Changelog
:cl:
add: Adds 18 new hairstyles/beards requested by the community.
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
Make bald 2 😏

**Comment by DrMacCool** [comment]
> 
> 
> Make bald 2 😏

Will you immediately merge the PR if I do?

**Comment by Archanial** [comment]
> 
> 
> > Make bald 2 smirk
> 
> Will you immediately merge the PR if I do?

yes

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -762,6 +802,10 @@
 	name = "Updo"
 	icon_state = "hair_updo"
 
+/datum/sprite_accessory/hair/vegeta
+	name = "Vegeta"
+	icon_state = "hair_vegeta"
```
Comment (Archanial) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]: I am unsure if we can have reference to Vegeta in code

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -824,6 +876,10 @@
 	name = "Beard (Cropped Fullbeard)"
 	icon_state = "facial_croppedfullbeard"
 
+/datum/sprite_accessory/facial_hair/gigachad
+	name = "Beard (Gigachad)"
+	icon_state = "facial_gigachad"
```
Comment (Archanial) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]: same here, not sure if we want to have reference to a meme, when those are banned in rules

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 1 above)
Comment (Kapu1178) [reply code/modules/mob/dead/new_player/sprite_accessories.dm]: Suggested rename: Pride

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 2 above)
Comment (DrMacCool) [reply code/modules/mob/dead/new_player/sprite_accessories.dm]: Yeah, I thought 'Gigachad' would be pushing the boundaries. Any suggestions for what would be better?

File: code/modules/mob/dead/new_player/sprite_accessories.dm (hunk 1 above)
Comment (DrMacCool) [reply code/modules/mob/dead/new_player/sprite_accessories.dm]: Gotcha.

**Comment by DrMacCool** [comment]
Jesus Christ GitHub is confusing. Ignore the metric shitton commits I just made.

Point is: Names changed, 'Bald 2' added (It's real).

I am going to blow my brains out if there's something else I've screwed up.

**Comment by Archanial** [comment]
It's fine github will squash it.

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 3]
```diff
@@ -790,6 +838,10 @@
 	name = "Wisp"
 	icon_state = "hair_wisp"
 
+
+
+
+
```
Comment (Crossedfall) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]:
```suggestion
```

All of this whitespace can go

**Comment by Crossedfall** [comment]
Can we get a single image with all the new hairstyles? It'd be much easier to see than looking at the sprite diffs. Or you can do sets of each and spoiler tag them under the correlating bullet.

**Comment by DrMacCool** [comment]
> 
> 
> Can we get a single image with all the new hairstyles? It'd be much easier to see than looking at the sprite diffs. Or you can do sets of each and spoiler tag them under the correlating bullet.

Sure thing. There's pictures of each of the styles that were requested over on the OG [forum thread](https://forums.beestation13.com/t/new-hairstyles/15009), but I can put them all here, too.

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**